demo/app.js

```diff
@@ -535,6 +547,18 @@ const Demo = React.createClass({
               title='This is the drawer component'
             >
               <div style={{ padding: 20, fontFamily: 'Helvetica, Arial, sans-serif' }}>Insert Custom Content Here</div>
+              <Button onClick={this._handleShowSmallDrawer}>
+                Toggle Small Drawer
+              </Button>
+              {this.state.showSmallDrawer ? (
+                <Drawer
+                  breakPoints={{ large: 750, medium: 500, small: 320 }}
```

Comment:
See [src/constants/Style.js#L2](https://github.com/mxenabled/mx-react-components/blob/0a62871140adbb6f8364025e67d4fe0c45791882/src/constants/Style.js#L2) for the defaults.